src/backend/binaryen/wasm_type_gen.ts

```diff
@@ -148,6 +149,9 @@ export class WASMTypeGen {
             case ValueTypeKind.OBJECT:
                 this.createWASMObjectType(<ObjectType>type);
                 break;
+            case ValueTypeKind.ENUM:
+                this.createWASMType((<EnumType>type).memberType);
```

Comment:
We can use a function `createWASMEnumType` here, so that the key in `typeMap` will be the enumType.

Reply:
copy